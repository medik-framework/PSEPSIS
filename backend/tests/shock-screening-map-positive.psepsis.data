{
  "Normal Saline": 45,
  "MAP": 50,
  "Epinephrine": 3,
  "Base Excess": -3,
  "Arterial Lactate": 340
}
