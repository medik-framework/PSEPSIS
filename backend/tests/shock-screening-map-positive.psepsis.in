[
  {
    "id": "tabletApp",
    "action": "broadcast",
    "eventName": "StartScreening",
    "eventArgs": []
  },
  {
    "id": "tabletApp",
    "action": "broadcast",
    "eventName": "AgeEntered",
    "eventArgs": [300],
    "waitFor":
     {
       "name": "Instruct",
       "args": ["get age"]
     }
  },
  {
    "id": "tabletApp",
    "action": "broadcast",
    "eventName": "SepsisSuspected",
    "eventArgs": [],
    "waitFor": {
      "name": "Instruct",
      "args": ["get weight"]
    }
  },
  {
    "action":"exit",
    "waitFor": {
      "id": "tabletApp",
      "interface": "TabletApp",
      "name": "ShockDiagnosis"
    }
  }
]
