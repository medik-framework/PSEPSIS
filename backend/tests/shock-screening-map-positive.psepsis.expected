[
  {
    "id": "tabletApp",
    "interface": "TabletApp",
    "name": "Instruct",
    "args": [
      "get age"
    ]
  },
  {
    "id": "tabletApp",
    "interface": "TabletApp",
    "name": "ConfirmAgeEntered",
    "args": []
  },
  {
    "id": "tabletApp",
    "interface": "TabletApp",
    "name": "Instruct",
    "args": [
      "get weight"
    ]
  },
  {
    "id": "datastore",
    "interface": "Datastore",
    "name": "Obtain",
    "args": [
      "Normal Saline"
    ]
  },
  {
    "id": "datastore",
    "interface": "Datastore",
    "name": "Obtain",
    "args": [
      "MAP"
    ]
  },
  {
    "id": "datastore",
    "interface": "Datastore",
    "name": "Obtain",
    "args": [
      "BP Sys"
    ]
  },
  {
    "id": "tabletApp",
    "interface": "TabletApp",
    "name": "ShockDiagnosis",
    "args": [true]
  }
]
